### ISSUE TITLE
Testing the maximum number of particles

### ISSUE THREAD

**Issue by henry2004y** [henry2004y/TestParticle.jl#58]
We would like to get a sense for a real-life case how many test particles we can trace simultaneously. This depends on how long we want to trace and how large the fields are, and we need to estimate for a practical situation.

**Comment by TCLiuu** [comment]
How to do this?

**Comment by henry2004y** [comment]
My idea is to use the current solution to #55 and then simply increase the number of particles until we run OOM. This will be more useful once #57 is finished such that we can then do some statistics.

**Comment by henry2004y** [comment]
This is not needed now since we know that the limiting factor is the size of the numerical field in most practical cases.
